Added body ID of index for better targeting with CSS

MatthewKennedy opened the pull request:
Could do with adding more unique ID's for catalog and other pages, too many instances of id="default"

MatthewKennedy committed:
Added body ID index for better targeting with CSS

Could do with adding more unique ID's for catalog and other pages, too many instances of id="default"

damianlegawiec commented:
Thanks @MatthewKennedy - can you supply more of those? :) Also I think for home it should be just `home`, `index` is kinda ambiguous

MatthewKennedy commented:
No problem, i’ll run through and add some unique ID’s for easier javascript and css targeting and change index to home-page


> On 25 Aug 2016, at 11:38, Damian Legawiec <notifications@github.com> wrote:
> 
> Thanks @MatthewKennedy <https://github.com/MatthewKennedy> - can you supply more of those? :) Also I think for home it should be just home, index is kinda ambiguous
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/spree/spree/pull/7545#issuecomment-242345246>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABLuvrb19ObBMk5unYBSe_bARnzj2BMRks5qjXCTgaJpZM4JpXFh>.
>

MatthewKennedy committed:
Added body ID's to home and checkout pages

Added checkout body id of ‘checkout-page’
Added/updated body id from ‘index’ to ‘home-page’

Other pages have unique ID’s
